internal/services/cdn/cdn_frontdoor_origin_resource.go

```diff
@@ -265,7 +267,7 @@ func resourceCdnFrontDoorOriginRead(d *pluginsdk.ResourceData, meta interface{})
 	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorOriginID(d.Id())
+	id, err := parse.FrontDoorOriginIDInsensitively(d.Id())
```

Comment:
this is coming from the Resource ID (which is being validated at import time) and so should be being parsed case-sensitively:

```suggestion
	id, err := parse.FrontDoorOriginID(d.Id())
```

Reply:
Fixed.